src/mathf/cubic-bezier.ts

```diff
@@ -75,6 +77,9 @@ import { Vector } from './vector';
  * @see https://stackoverflow.com/questions/27053888/how-to-get-time-value-from-bezier-curve-given-length
  */
 export class CubicBezier {
+    static EASE_IN_OUT_SINE: EasingFunction =
```

Comment:
Do we need this?

Reply:
No, I thought it might be handy to start adding cheat-sheet style constants, but I can move this inline to where it's used.